domain/host/host.go

```diff
@@ -188,13 +193,26 @@ func Build(ip string, rpcport string, poolid string, memory string, ipAddrs ...s
 	}
 	host, err := currentHost(ip, rpcPort, poolid)
 	if err != nil {
-		glog.V(2).Infof("currentHost failed: %s", err)
+		plog.WithError(err).WithFields(log.Fields{
+			"ip": ip,
+			"rpcPort": rpcPort,
+			"poolID": poolid,
```

Comment:
The convention for field names we've been going with so far is all lowercase. Reduces clutter. That said, we could undo that convention if you feel strongly in the other direction.